### ISSUE TITLE
Final clean-up work for 0.0.1

### ISSUE THREAD

**Issue by krsyoung** [harled/matey#35]
Finalize clean-up work for a 0.1.1 gem push to rubygems:

- [x] update authors
- [x] update emails
- [x] separate test and linting CI steps
- [x] fix gem build errors

**Comment by krsyoung** [comment]
gem build errors

```
vscode ➜ /workspaces/matey (main ✗) $ gem build matey.gemspec 
WARNING:  open-ended dependency on view_component (>= 0) is not recommended
  use a bounded requirement, such as '~> x.y'
WARNING:  open-ended dependency on ahoy_matey (>= 0) is not recommended
  use a bounded requirement, such as '~> x.y'
WARNING:  open-ended dependency on factory_bot (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'
WARNING:  open-ended dependency on sqlite3 (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'
WARNING:  open-ended dependency on debug (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'
WARNING:  See https://guides.rubygems.org/specification-reference/ for help
  Successfully built RubyGem
  Name: matey
  Version: 0.1.1
  File: matey-0.1.1.gem
```

**Comment by krsyoung** [comment]
Closed via #36
